path_login_test.go

```diff
@@ -952,7 +939,7 @@ func TestLoginProjectedToken(t *testing.T) {
 			role:        "plugin-test",
 			jwt:         jwtProjectedDataExpired,
 			tokenReview: testProjectedMockFactory,
-			e:           errors.New("token is expired"), // jwt.ErrTokenIsExpired,
+			e:           errors.New("Token is expired"),
```

Comment:
I concur, but then the test fails, because somewhere the error gets transformed into starting with an uppercase letter. I didn't immediately see where the conversion was happening in the `jwt` library.

(This is why I switched to `strings.ToLower()` elsewhere.)

Reply:
😱 Somehow I missed that this was test data on my previous review, how weird/annoying